ignition/src/GazeboWrapper.cpp

```diff
@@ -38,6 +38,9 @@
 #include <sdf/Root.hh>
 #include <sdf/World.hh>
 
+#include <ignition/fuel_tools/FuelClient.hh>
+#include <ignition/fuel_tools/Interface.hh>
+
```

Comment:
These headers have been already included in the header file. Though, since they should not be in the header file in the first place, this is their right location.